### PR TITLE
Update ppp

### PR DESCRIPTION
**Pull request by ShawnChew**

### CODE REVIEW

File: docs/team/shawnchew.md [hunk 1]
```diff
@@ -7,31 +7,37 @@ title: Shawn Chew's Project Portfolio Page
...
     * Developer Guide:
-        * To be added
+        * Added implementation details for view commands. (Pull Request [#96](https://github.com/AY2223S1-CS2103T-T17-3/tp/pull/96))
 
 * **Community**:
-    * To be added
-
-* **Tools**:
-    * To be added
+    * Reviewed 15 Pull Requests on [GitHub](https://github.com/AY2223S1-CS2103T-T17-3/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3A%40me).
```
Comment (lesterong) [inline docs/team/shawnchew.md:40]: Maybe you want to update this link? the `reviewed-by` is currently set to `@me`, which means it'll be set to whichever user that's logged in at the moment.

File: docs/team/shawnchew.md [hunk 2]
```diff
@@ -7,31 +7,37 @@ title: Shawn Chew's Project Portfolio Page
...
+    * Found 13 bugs in team [CS2103-T10-3](https://github.com/shawnchew/ped)'s product during PE-D.
```
Comment (lesterong) [inline docs/team/shawnchew.md:41]: Minor, but I think the team is CS2103T-T10-3

Comment (lesterong) [inline docs/team/shawnchew.md:41]: Minor, but I think the team is CS2103T-T10-3.

**Review by lesterong — APPROVED** [review]
Just some minor things you might want to update, otherwise, you can just merge it yourself!

File: docs/team/shawnchew.md (hunk 1 above)
Comment (ShawnChew) [reply docs/team/shawnchew.md:40]: Okay thanks!

Comment (lesterong) [reply docs/team/shawnchew.md]: Hey @ShawnChew, it seems like the updated link is still incorrect, do remember to update it in the final PPP you're going to submit. Your new link just shows all the PRs excluding PRs authored by you. 

You might want to use `https://github.com/AY2223S1-CS2103T-T17-3/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3AShawnChew` or `https://github.com/AY2223S1-CS2103T-T17-3/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3AShawnChew+-author%3AShawnChew`. 

The former includes PRs authored by you while the latter does not.

**Pull request merged**